workflow.src

```diff
@@ -0,0 +1,121 @@
+        It lets you tell others about changes you've pushed to the repository.
+        Once a pull request is opened, you can discuss and review the potential changes with
+        others and add follow-up commits before the changes are merged into the repository.
+      </p>
+  <ol>
+    <li>
+      <p>
+        Navigate to the repo.
+      </p>
+      <pre><code>cd specs-on-github</code></pre>
+    </li>
+    <li>
+      <p>
+        Checkout the main branch of your repository (in general, this is the <code>master</code> branch).
+      </p>
+      <pre><code>git checkout master</code></pre>
+    </li>
+    <li>
+      <p>
+        Make sure the main branch is up-to-date with upstream.
+      </p>
+      <pre><code>git pull</code></pre>
+    </li>
+    <li>
+      <p>
+        Create a new branch for your upcoming pull request (one branch per pull request). Please make branch names
+        informative - by including the issue or bug number for example.
+      </p>
+      <pre><code>git checkout -B my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Make your edits in your favorite editor.
+      </p>
+    </li>
+    <li>
+      <p>
+        Add your edits to prepare your commit by staging them. Best is to use the interactive mode (New files still
+        need to be added explictly).
+      </p>
+      <pre><code>git add -p</code></pre>
+      <p>If you ignore an edit, it will be reproposed to you the next time you run the command. This allow you to make
+      multiple edits at once and split them into different more meaningfull/easy-to-review commits.</p>
+    </li>
+    <li>
+      <p>
+        Commit your staged edits. If you're fixing an issue, reference it in the commit e.g. <code>#87</code>
+        (it will automatically close the issue once the commit is added in the main branch).
+      </p>
+      <pre><code>git commit -m "&lt;purpose of your edits>"</code></pre>
+    </li>
+    <li>
+      <p>
+        Once all your commits are done, push your branch upstream.
+      </p>
+      <pre><code> git push --set-upstream origin my-wonderful-edits-for-87</code></pre>
+    </li>
+    <li>
+      <p>
+        Using your favorite browser, navigate to the repo on GitHub, e.g.
+      </p>
+      <pre><code>https://github.com/tobie/specs-on-github/</code></pre>
+    </li>
+    <li>
+      <p>
+        Once your anthenticated on the repo page, a new button should appear on the page to create a new
+        pull request (<code>Compare &amp; pull request</code>). Go ahead and punch it.
+      </p>
+      <p><img src="img/create-a-pr.png" alt="Compare &amp; pull request" /></p>
+    </li>
+    <li>
+      <p>
+        Make sure you like the title of your pull request, add a proper description, and use
+        <a href="spec-labels.html">proper GitHub labels</a>, add one or more reviewers so they get pinged.
```

Comment:
The document should mention something about continuing to send commits until all review comments have been addressed.

Reply:
I expanded a bit later on at the review part.